src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -134,6 +134,9 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
 
   /** Boolean value to indicate if the role application should be applied on details list. Set to false by default */
   shouldApplyApplicationRole?: boolean;
+
+  /** Callback for selection zone to indicate whether the item was selected */
+  onItemSelected?: (isSelected: boolean) => void;
```

Comment:
We wanted to say something like this read “File Name, File Type, selected” , aria-selected doesn't seem to serve the whole purpose.